corehq/apps/tzmigration/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+
+
+class MigrationStatus(object):
+    NOT_STARTED = 'not_started'
+    IN_PROGRESS = 'in_progress'
+    COMPLETE = 'complete'
+
+    choices = [
+        (NOT_STARTED, 'Not Started'),
+        (IN_PROGRESS, 'In Progress'),
+        (COMPLETE, 'Complete'),
+    ]
+
+    max_length = max(len(choice) for choice, _ in choices)
+
+
+class TimezoneMigrationProgress(models.Model):
+    """
+    The 2015 timezone migration will happen on a per-domain basis
+    This set of "toggles" and associated info
+    will be used for the following purposes:
+    - to determine how reports, etc. will interpret datetimes
+      in form and case data
+    - whether to use updated "correct" tz behavior
+      or to emulate previous "incorrect" tz behavior when processing forms
+    - to track progress of the migration of a domain, during the migration
+
+    """
+    domain = models.CharField(max_length=256, null=False, blank=False,
+                              db_index=True, primary_key=True)
+    migration_status = models.CharField(choices=MigrationStatus.choices,
+                                        max_length=MigrationStatus.max_length,
+                                        default=MigrationStatus.NOT_STARTED)
```

Comment:
Yeah, I'm just not really sure in this case because I may also want to record progress through the migration (number of forms migrated, number of cases migrated, maybe a checkpoint of some sort depending on how we do the migration, etc.). Anyways, I didn't want to engineer a general solution for a single case when I'm not actually totally sure how-all I'm going to use it.